src/bot.ml

```diff
@@ -57,10 +57,13 @@ let callback _conn req body =
     if
       string_match
         ~regexp:
-          ( f "@%s:? [Mm]inimize[^`]*```[^\n]*\n\\(\\(.\\|\n\\)+\\)"
+          ( f "@%s:? [Mm]inimize\\([^`]*\\)```[^\n]*\n\\(\\(.\\|\n\\)+\\)"
           @@ Str.quote bot_name )
         body
-    then Some (Str.matched_group 1 body |> extract_minimize_file)
+    then
+      Some
+        ( Str.matched_group 1 body
+        , Str.matched_group 2 body |> extract_minimize_file )
```

Comment:
The issue seems to be this pair of lines.  Maybe `extract_minimize_file` messes things up in a way that `Str.matched_group 1 body` fails? (Does OCaml evaluate tuples right-to-left?)

Reply:
tuple evaulation order is undefined afaik